src/main/kotlin/fi/metatavu/muisti/pages/PageLayoutController.kt

```diff
@@ -45,6 +45,36 @@ class PageLayoutController() {
         return pageLayoutDAO.findById(id)
     }
 
+    /**
+     * Finds a list of exhibition page layouts by device model id and screen orientation
+     *
+     * @param deviceModelId device model id
+     * @param screenOrientation screen orientation
+     * @return list of exhibition page layouts
+     */
+    fun findPageLayoutsByDeviceModelIdAndOrientation(deviceModelId: UUID, screenOrientation: ScreenOrientation): List<PageLayout> {
```

Comment:
Complex list methods should be merged into one list method. See example from ExhibitionDeviceController::listExhibitionDevices